lib/MusicBrainz/Server/Controller/Edit.pm

```diff
@@ -337,8 +337,7 @@ sub edit_type : Path('/doc/Edit_Types') Args(1) {
     my ($self, $c, $edit_type) = @_;
 
     my $class = EditRegistry->class_from_type($edit_type);
-    my $id = 'Edit Type/$class->edit_name';
-    $id =~ s/ /_/g;
+    my $id = 'Edit Type/$class->edit_name' =~ tr/ /_/r;
```

Comment:
Yeah, good catch. This went in in 2011, so it seems nobody ever tried to add further information on edit types via the wiki. The delayed check whether `$class` actually exists doesn’t look right, either.

(And BTW, I don’t think you can do method calls inside a double-quote string even.)